css/general.css

```diff
@@ -105,77 +112,192 @@
     margin: 1px;
   }
 
-  .cards {
-    display: flex;
-    flex-direction: column;
-    margin: 30px 15%;
-    width: 70%;
-    align-items: center;
-    gap: 80px;
+  .card-works {
+    display: grid;
+    grid-gap: 5em;
+    background-color: #e5e5e5;
+    justify-content: center;
   }
 
   .card {
     display: flex;
-    width: 330px;
     flex-direction: row;
-    border: 1px solid rgb(223 220 220);
-    border-radius: 15px;
-    height: 550px;
-    background-color: #fff;
+    justify-content: center;
+    background: #fff;
+    border: 2px solid #a7aeff;
+    border-radius: 24px;
+    width: 75vw;
+    height: 58vh;
+    gap: 2em;
+  }
+
+  .card1 {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: center;
+    background: #fff;
+    border: 2px solid #a7aeff;
+    border-radius: 24px;
+    width: 75vw;
+    height: 58vh;
+    gap: -1em;
+  }
+
+  .card-image {
+    border-radius: 8px;
+    background-repeat: no-repeat;
+    background-size: 100% 78%;
+    margin-left: 10px;
+    margin-top: 20px;
+    width: 580px;
+    height: 500px;
+  }
+
+  .card-image1 {
+    background-image: url("../image/Snap3.png");
+  }
+
+  .card-image2 {
+    background-image: url("../image/Snap2.png");
+    margin-right: 30px;
```

Comment:
- [ ] As suggested by the previous reviewer, You should implement your code to avoid child elements overflow. I noticed you tried to fix that issues but it broke some points and your image and text inside your cards have exceeded their parent card div. Please fix these issues to make your design match the Figma template.
### Hint:

- I saw that you used all card images as `background-image`. However, to avoid these issues, It's good to add them as HTML elements using the `<img>` tag and use either `flexbox` or `grid` to align them with their sibling's elements.
After that, you can use the CSS properties `object-fit: cover` and `overflow: hidden` to ensure that your image is well fixed in its container.

**1. Your cards**
![image](https://user-images.githubusercontent.com/80612925/171937715-221b4efa-2b5d-4120-bd29-2c3ec03f0ee7.png)

**2. The Figma desing**
![image](https://user-images.githubusercontent.com/80612925/171937573-8d81ad2c-c807-4399-9326-639ef96a7a4c.png)